cockatrice/src/client/tabs/tab_game.cpp

```diff
@@ -1699,12 +1711,33 @@ void TabGame::createPlayAreaWidget(bool bReplay)
 
 void TabGame::createReplayDock()
 {
+    // timeline widget
```

Comment:
Is this comment needed?

Reply:
Probably not. But since I decided to put comments at the top of every grouping, I felt like it looks nicer if there's something on this line.